src/SiteHeader.js

```diff
@@ -0,0 +1,225 @@
+                    key={'link-' + index}
+                    {...item} 
+                  />
+                );
+
+              }
+
+            }, this)
+          }</nav>
+      </div>
+    )
+  }
+
+  renderDesktopNav() {
+    return (
+      <header className="site-header desktop">
+        <div className="container-fluid">
+        <div className="nav-container d-flex align-items-end">
+          
+          <div className="brand">
+            {this.renderLogo()}
+          </div>
+          
+          <div className="flex-grow-1 d-flex flex-column-reverse">
+
+            <div className="nav primary-menu-container">
+              {
+                this.renderNav(this.props.desktopMenuItems, {
+                  menuClassName: "primary-menu",
+                  triggerClassName: "nav-link",
+                  respondToPointerEvents: true
+                }, {
+                  className: "nav-link"
+                })
+              }
+            </div>
+
+            <div className="nav align-self-end secondary-menu-container align-items-start mb-4 mt-3">
+              <form className="form-inline">
+                <div className="input-group">
+                  <input type="text" className="form-control form-control-sm" placeholder="Search for subjects and more..." />
+                </div>
+              </form>
+              {
+                this.renderNav(this.props.secondaryMenuItems, {
+                  menuClassName: "secondary-menu",
+                  triggerClassName: "btn btn-sm ml-sm-2",
+                  respondToPointerEvents: false
+                }, {
+                  className: "btn btn-sm ml-sm-2"
+                })
+              }
+            </div>
+          </div>
+        </div>
+        </div>
+      </header>
+    );
+  }
+
+  renderNav(navItems, menuOptions, linkOptions) {
+    return navItems.map(function(item, index){
+      if (item.submenu) {
+        return (
+          <Menu 
+            key={"menu-" + index}
+            className={menuOptions.className}
+            triggerClassName={classNames(menuOptions.triggerClassName, item.triggerClassName)}
+            triggerContent={<span>{item.content} <FontAwesomeIcon icon="chevron-down" /></span>}
+            triggerDestination={item.destination}
+            respondToPointerEvents={menuOptions.respondToPointerEvents}
+          >
+            {item.submenu}
+          </Menu>
+        );
+      } else {
+        return (
+          <Hyperlink 
+            key={'link-' + index}
+            className={classNames(linkOptions.className, item.triggerClassName)}
+            {...item} 
+          />
+        )
+      }
+    },this)
+  }
+
+  renderLogo() {
+    return (
+      <Hyperlink 
+        className="header-logo" 
+        content={<img src={this.props.logo} alt={this.props.logoAltText} />} 
+        destination={this.props.logoDestination}
+      />
+    )
+  }
+
+  renderAccountMenuContent() {
```

Comment:
We'll need to figure out a way to make the content rendered here configurable based on the logged in status of the user.